### ISSUE TITLE
Every 2nd Tuesday kinds of meetings.

### ISSUE THREAD

**Issue by fordcrews** [williamtroup/Calendar.js#71]
Is there a way to have a occurring meeting that happens every 2nd Tuesday, 3rd Wednesday, kind of things

**Comment by williamtroup** [comment]
Yes, it's called a Repeating event.  When adding an event, there is a tab called "Repeats".  Everything you need is under there.

Please star and share if you like the project!

Thanks for using Calender.js.

**Comment by fordcrews** [comment]
Maybe I'm confused, thanks for the quick reply.   I tried repeat, went on a Wednesday, 11th of may, picked custom and repeat monthly, and it's the 11th of every month.  I tried custom and monthly, still 11th of each month, excluded every day but Wednesday, and I got a repeating appointment every month where the 11th was on a Wednesday.

**Comment by williamtroup** [comment]
Under the "Repeats" tab, select "Custom", select "Daily" and set 7 in the input box for repeating on that day every week.  I will let you determine what you need from that.

Thanks
